From 6a931ca559a937cb9f7917b9131ec1aca4919683 Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Mon, 26 Sep 2022 05:23:06 +0100
Subject: [PATCH] acinclude.m4: fix typo in 'x' if check

Case doesn't matter but case does need to match on left & right
hand side, so just choose uppercase everywhere to match what
the rest of the file does.
---
 acinclude.m4 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/acinclude.m4 b/acinclude.m4
index cc21ff6..7d77f01 100644
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -278,7 +278,7 @@ dnl in <features.h>. We don't use __GNUC_PREREQ directly because <features.h>
 dnl is not present on all the operating systems that we support, e.g. OpenBSD.
 dnl
 AC_DEFUN([GCC_FORTIFY_SOURCE],[
-   if test "x$CC" != "X"; then
+   if test "X$CC" != "X"; then
       AC_MSG_CHECKING([whether ${CC} accepts -D_FORTIFY_SOURCE])
       AC_TRY_COMPILE(,[
          #define GNUC_PREREQ(maj, min) ((__GNUC__ << 16) + __GNUC_MINOR__ >= ((maj) << 16) + (min))
@@ -311,7 +311,7 @@ dnl If it is not supported, then the test program will compile without
 dnl warnings.
 dnl
 AC_DEFUN([GCC_FORMAT_SECURITY],[
-   if test "x$CC" != "X"; then
+   if test "X$CC" != "X"; then
       AC_MSG_CHECKING([whether ${CC} accepts -Wformat-security])
       wfs_old_cflags="$CFLAGS"
       CFLAGS="$CFLAGS -Wall -Werror -Wformat -Wformat-security"
-- 
2.43.0

